adapter/service/parameters/model.go

```diff
@@ -0,0 +1,66 @@
+package parameters
+
+const (
+	WidgetTypeInput       = "input"
+	WidgetTypeSelect      = "select"
+	WidgetTypeMultiSelect = "multiselect"
+
+	ValueTypeInt         = "int"
+	ValueTypeString      = "string"
+	ValueTypeBool        = "bool"
+	ValueTypeIntArray    = "int_array"
+	ValueTypeStringArray = "string_array"
+)
+
+// TODO: add helper methods
+type SupportedParameter struct {
+	ID           string         `json:"parameter_id"`
+	Name         string         `json:"name"`
+	Description  string         `json:"description"`
+	ValueType    string         `json:"value_type"`
```

Comment:
Define enums for value type and widget type.

Reply:
I mean type of enums.

Reply:
Done.